### PR TITLE
Fixed #999: 'Hound CI integration'

### LINKED ISSUE #999: Hound CI integration
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [X] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**

**Expected behavior:**

To keep a track on proper style of code.

**Steps to reproduce:**

**Related code:**

```
insert any relevant code here else remove this section
```

**Other information:**

Hound comments on style violations in PRs, allowing us to better review and maintain a clean codebase.

Reference: https://houndci.com/
**System information:** 

```
Your operating system
```

```
output of `node -v`
```

```
output of `npm ls ember-cli`
```

### PR DESCRIPTION
**Pull request by simsausaurabh**
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This integrates Hound CI to check the code style of the project.

#### Changes proposed in this pull request:

- Integrated Hound CI
-
-

Fixes #999

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-1021-open-event-frontend-fossasia.surge.sh

**Comment by simsausaurabh** [comment]
@mariobehling @niranjan94 @CosmicCoder96  @geekyd @sumedh123 Please review, thanks.

**Comment by niranjan94** [comment]
Deployment link: https://pr-1021-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-1021-open-event-frontend-fossasia.surge.sh

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+jshint:
+  enabled: true
+  config_file: app/.jshintrc
+  ignore_file: .javascript_ignore
```
Comment (ParthS007) [inline .hound.yml]: Add a line at EOF

File: .javascript_ignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+dist/
+bower_components/
```
Comment (ParthS007) [inline .javascript_ignore]: same

File: app/.jshintrc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	"scripturl": false,
+	"shadow": false,
+	"sub": true,
+	"supernew": false,
+	"validthis": true,
+	"noyield": false,
+	"browser": true,
+	"devel": false,
+	"node": false,
+}
```
Comment (ParthS007) [inline app/.jshintrc]: same

**Comment by niranjan94** [comment]
Deployment link: https://pr-1021-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-1021-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-1021-open-event-frontend-fossasia.surge.sh

**Comment by simsausaurabh** [comment]
@ParthS007 please review the changes made as requested.

**Comment by simsausaurabh** [comment]
@ParthS007 An extra line at the end of each file has been inserted, I think its because the changes you requested where on the previous line and without modifying that line - an extra line has been added below it.

**Comment by CosmicCoder96** [comment]
@simsausaurabh ES-lint already takes care of the any lint issues that may arise, closing this PR as per discussion with @niranjan94 as well.

**Comment by simsausaurabh** [comment]
@CosmicCoder96 Thank you for reviewing. And your point is totally valid, Since there is ```.eslintrc.js``` already in the repo, there is no point in having HoundCI.

**Pull request closed without merging**